src/rgw/rgw_lc.cc

```diff
@@ -1695,9 +1695,14 @@ int RGWLC::bucket_lc_process(string& shard_id, LCWorker* worker,
     return -ENOENT;
   }
 
-  map<string, bufferlist>::iterator aiter = bucket->get_attrs().find(RGW_ATTR_LC);
-  if (aiter == bucket->get_attrs().end())
+  map<string, bufferlist>::iterator aiter
+    = bucket->get_attrs().find(RGW_ATTR_LC);
+  if (aiter == bucket->get_attrs().end()) {
+    ldpp_dout(this, 0) << "bucket_attrs.find(RGW_ATTR_LC) failed for "
```

Comment:
Don't know how warning-ish this is, but consider prefixing log message with "WARNING: ".

Reply:
can do